<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-12T19:50:31Z" name="Default suite" finished-at="2022-08-12T19:51:05Z" duration-ms="33453">
    <groups>
    </groups>
    <test started-at="2022-08-12T19:50:31Z" name="Default test" finished-at="2022-08-12T19:51:05Z" duration-ms="33453">
      <class name="com.java.pom.TestPage">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:com.java.pom.TestPage@76a4ebf2]" started-at="2022-08-13T01:20:31Z" name="launchBrowser" finished-at="2022-08-13T01:20:41Z" duration-ms="10040" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="signIn()[pri:0, instance:com.java.pom.TestPage@76a4ebf2]" started-at="2022-08-13T01:20:41Z" name="signIn" finished-at="2022-08-13T01:21:05Z" duration-ms="23251" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [[[ChromeDriver: chrome on WINDOWS (c05248580bfd34e1685610dff37af0f2)] -> id: total_product]] but found [45.38]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [[[ChromeDriver: chrome on WINDOWS (c05248580bfd34e1685610dff37af0f2)] -> id: total_product]] but found [45.38]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at com.java.pom.TestPage.signIn(TestPage.java:115)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method is-config="true" signature="terminateBrowser()[pri:0, instance:com.java.pom.TestPage@76a4ebf2]" started-at="2022-08-13T01:21:05Z" name="terminateBrowser" finished-at="2022-08-13T01:21:05Z" duration-ms="136" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- terminateBrowser -->
      </class> <!-- com.java.pom.TestPage -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
